src/devices/bus/megadrive/svp.cpp

```diff
@@ -54,7 +74,7 @@ md_rom_svp_device::md_rom_svp_device(const machine_config &mconfig, const char *
 
 ROM_START( svp )
 	ROM_REGION(0x800, "internal_rom", 0)
```

Comment:
Thanks for the PR!
If you do this you also need to update the size of the ROM_REGION from 0x800 to 0x20000.

Reply:
@Osso13 thanks for the tip! I was looking at both functions headers but I wasn't really sure on how to use them. This won't affect the loading of the actual game code now that the ROM_REGION for the internal rom is occupying the whole SVP code space, right?

Reply:
Well, I think this needs more work before it can be used.
The preferred way, I believe, would be to leave it as it was before this PR and then map it in the memory_map like

map(0x1f800, 0x1ffff).rom().region("internal_rom", 0);

But as said the point for now is just to have MAME load the ROM to preserve it, it will need more work before it's correctly used by the emulation.

Reply:
@Osso13 I was thinking that too. Besides reverting the code part back, do you think the comment I added reflects what you mention clearly? Or should I add a `FIXME` comment in the specific area with what you mentioned in the comment? Let me know and I can push a commit for that. Thanks!

Reply:
My opinion is the notes you added already explain very well where the ROM is seen by the CPU, so I'd just leave it at that. But before acting please wait and see if other, better devs have different opinions.
Thanks!

Reply:
Yup, the CPU device should be able to map it to the correct offset, no real need to do it here.

Reply:
@happppp thanks, I just removed the code changes considering that additional work is required for this to work properly. I don't know if I could get at it at some point (not really experienced with the code base) but I can help verifying the real behavior of the chip compared to the emulation in the future if the team requires it :).

Please let me know if the tech notes I added are sufficient to note which addresses the internal ROM should be mapped to. Thanks!!

Reply:
Your notes are clear =)